### PR TITLE
Implement week 6 subjects

### PR DESCRIPTION
**Pull request by HosseinDinarvand**
I apply bellow changes:
1) add .gitignore file
2) update middle pipline in program.cs
3) change folder structure
4) apply another changes

### CODE REVIEW

**Comment by VahidFarahmandian** [comment]
> I apply bellow changes:
> 
> 1. add .gitignore file
> 2. update middle pipline in program.cs
> 3. change folder structure
> 4. apply another changes

@HosseinDinarvand you have placed gitignore file in the root of the repository which will effect other folders that you do not own. You should move it to your own folder so that it will effect only your files/folders.

**Comment by VahidFarahmandian** [comment]
If you are opening a PR regarding an issue or set of issues, you need to specify them in Development section if the PR. 
I did it for you and you can check it.

**Comment by HosseinDinarvand** [comment]
> > I apply bellow changes:
> > 
> > 1. add .gitignore file
> > 2. update middle pipline in program.cs
> > 3. change folder structure
> > 4. apply another changes
> 
> @HosseinDinarvand you have placed gitignore file in the root of the repository which will effect other folders that you do not own. You should move it to your own folder so that it will effect only your files/folders.


OK, I will add .gitignore in my own folder and push again.

**Comment by HosseinDinarvand** [comment]
> If you are opening a PR regarding an issue or set of issues, you need to specify them in Development section of the PR. I did it for you and you can check it.

Ok,I will follow this recommendation in the next issues.
thank's for your help.

**Review by VahidFarahmandian — CHANGES_REQUESTED** [review]
gitignore :-|

File: .gitignore
Comment (VahidFarahmandian) [inline .gitignore:1]: this file should be removed.

**Review by VahidFarahmandian — CHANGES_REQUESTED** [review]

**Comment by HosseinDinarvand** [comment]
I add **MsTest** project for ConsoleApp by using **Moq package** in last commit, please check my code.

**Review by VahidFarahmandian — APPROVED** [review]

**Pull request merged**